pkg/knowledge_base/edge_kb.go

```diff
@@ -56,14 +56,17 @@ type (
 		Source core.Resource
 		// Destination refers to the initial target resource node when edge expansion is called
 		Destination core.Resource
+		// Routes refers to any api routes being satisfied by the edge
+		Routes []core.Route
```

Comment:
so the other thought is we create an integration for each route and then attach that to the lambda. Otherwise, not quite sure how we would get around that, but i also think its valid for a an edge of GW -> Lambda to have data of routes